modules/compliance-increasing-operator-limits.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-increasing-operator-limits"]
+= Increasing Compliance Operator resource limits
+
+If the Operator has been installed through OLM, the Compliance Operator might require more memory than the default limits allow. The best way to mitigate this issue is to set custom resource limits.
```

Comment:
So this is tricky. You're right that in many cases the memory issues were caused by OLM's way of installing operators that watch all namespaces by default. But the memory issues are not all caused strictly by this, so I wonder if we should rephrase the first sentence to say something more generic like "In some cases, the Compliance Operator might require more memory than the default limits allow".